tests/interfaces/network.go

```diff
@@ -0,0 +1,52 @@
+package interfaces
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math/big"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/subnet-evm/ethclient"
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/TeleporterMessenger"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type SubnetTestInfo struct {
```

Comment:
nit: Since this type is used often, I'd prefer to have it in its own file in the same package.

Reply:
Good point, and probably not worth adding a whole other package just for this type. I'm cool leaving this here.

Reply:
I actually went ahead and moved it. If anyone feels strongly let me know, but I'll leave it as is now otherwise.